Vagrantfile

```diff
@@ -5,6 +5,17 @@ def yellow(text); colorize(text, "\033[33m"); end
 def green(text); colorize(text, "\033[32m"); end
 def bold(text); colorize(text, "\033[1;97m"); end
 
+required_plugins = ['vagrant-vbguest', 'vagrant-hostmanager']
+plugins_to_install = required_plugins.select { |plugin| !Vagrant.has_plugin? plugin }
+unless plugins_to_install.empty?
+  puts "Installing plugins: #{plugins_to_install.join(', ')}"
+  if system "vagrant plugin install #{plugins_to_install.join(' ')}"
```

Comment:
Please refer to the comment there: https://github.com/spryker/devvm/pull/26/commits/b34f03c61b7ad54c349ddc4b9e71156d4d4ffb6c#diff-23b6f443c01ea2efcb4f36eedfea9089R15